azext_iot/central/command_map.py

```diff
@@ -47,13 +47,48 @@
     operations_tmpl="azext_iot.central.commands_api_token#{}"
 )
 
+central_query_ops = CliCommandType(
+    operations_tmpl="azext_iot.central.commands_query#{}"
+)
+
+central_destination_ops = CliCommandType(
+    operations_tmpl="azext_iot.central.commands_destination#{}"
+)
+
+central_export_ops = CliCommandType(
+    operations_tmpl="azext_iot.central.commands_export#{}"
+)
+
 
 # Dev note - think of this as the "router" and all self.command_group as the controllers
 def load_central_commands(self, _):
     """
     Load CLI commands
     """
 
+    with self.command_group(
+        "iot central", command_type=central_query_ops, is_preview=True
+    ) as cmd_group:
+        cmd_group.command("query", "query_run")
+
+    with self.command_group(
+        "iot central destination", command_type=central_destination_ops, is_preview=True
```

Comment:
Have you considered `[az] iot central export destination` i.e. `destination` to be a command group under `export` rather than a sibling? Saying this if destinations are specific for data export and nothing else. This way, all elements related to IoT Central data export are anchored under `az iot central export`.

Reply:
That's a good suggestion, destination can't be used separately without export, I have re-grouped it under export as suggest.